torch/_inductor/ir.py

```diff
@@ -998,11 +998,11 @@ def as_storage_and_layout(x, freeze=True, want_contiguous=False, stride_order=No
                 x.data.decide_layout()
         return x, x.data.layout
     if isinstance(x, ReinterpretView):
+        # making the base of x contiguous or stride_ordered will not necessarily make
+        # the ReinterpretedView either, so dont pass along those arguments
```

Comment:
Ignoring those args seems fine for current usages, but it still feels fragile (nothing to be done in this PR however)